app/metrics/server/lib/collectMetrics.js

```diff
@@ -31,6 +31,16 @@ const setPrometheusData = async () => {
 	metrics.ddpAuthenticatedSessions.set(authenticatedSessions.length);
 	metrics.ddpConnectedUsers.set(_.unique(authenticatedSessions.map((s) => s.userId)).length);
 
+	// Apps metrics
```

Comment:
moved this up because it was not running if no statistics were found https://github.com/RocketChat/Rocket.Chat/pull/20832/files#diff-d5557cb19a93c53e1e732cf0f6ca3ef772654b5a9fc40fa9f92092f6d304d196R45